python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py

```diff
@@ -338,6 +346,7 @@ def _dbt_nodes_to_assets(
         ins=dict(asset_ins.values()),
```

Comment:
git won't let me comment on the correct line, but I think technically the `op_name` parameter should be a hash of both the select AND the exclude parameters. I think in practice, this will not cause problems (because you'd need to have multiple invocations of load_assets...() with the same select parameter but different exclude parameters, which would be hard to do in any valid workflow, but I thought I'd mention.